server/graphql/v2/mutation/MemberMutations.js

```diff
@@ -145,22 +145,24 @@ const memberMutations = {
         }
       }
 
-      // Edit member
-      const editableAttributes = pick(args, ['role', 'description', 'since']);
-
-      const [, members] = await models.Member.update(editableAttributes, {
-        returning: true,
+      const member = await models.Member.findOne({
         where: {
           MemberCollectiveId: memberAccount.id,
           CollectiveId: account.id,
+          role: [MemberRoles.ACCOUNTANT, MemberRoles.ADMIN, MemberRoles.MEMBER],
```

Comment:
Can we keep only this line? It seems that the other changes (`findOne` + `update`) are not necessary

Reply:
@Betree yes, tested here and it worked.

Reply:
Updated